lib/optimizely/custom_attribute_condition_evaluator.rb

```diff
@@ -327,6 +330,25 @@ def semver_less_than_or_equal_evaluator(condition)
       SemanticVersion.compare_user_version_with_target_version(target_version, user_version) <= 0
     end
 
+    def qualified_evaluator(condition)
+      # Evaluate the given match condition for the given user qaulified segments.
+      # Returns boolean true if condition value is in the user's qualified segments,
+      #                 false if the condition value is not in the user's qualified segments,
```

Comment:
Is this wide indent a ruby commenting style? I don't know, just mentioning :)
Seems a bit weird?

Reply:
I just matched the existing style in this file. Looks like the intention was to line up the different return values visually. I have no strong feelings about it either way :shrug: